src/s2client-fsharp/Instance.fs

```diff
@@ -93,7 +100,8 @@ module Instance =
 
     let createGame (instance:Sc2Instance) mapName (participants:Participant list) realTime = async {
         let req = new RequestCreateGame()
-        for player in participants do
+        participants
+        |> List.iter (fun player ->
```

Comment:
I'm not sure if List.iter is that much better than just using a for loop...

Reply:
I don't mind either way, happy to revert.